### PR TITLE
Add UI for password change form

### PR DESCRIPTION
**Pull request by Mloweedgar**

### CODE REVIEW

File: src/Auth/components/ChangePassword/index.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import React from 'react';
```
Comment (BenMaruchu) [inline src/Auth/components/ChangePassword/index.js:1]: I think we should add user input for current password so as improve security if someone is using your PC he/she shouldn't be able to change your password

Comment (BenMaruchu) [reply src/Auth/components/ChangePassword/index.js:1]: What do you think @Mloweedgar ?

Comment (Mloweedgar) [reply src/Auth/components/ChangePassword/index.js:1]: yes, Let me add it

Comment (BenMaruchu) [reply src/Auth/components/ChangePassword/index.js:1]: Is it done?

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**